src/plugins/discover/public/application/components/data_grid/data_grid_table_flyout.tsx

```diff
@@ -52,7 +52,9 @@ export function DataGridFlyout({
       <EuiFlyoutBody>
         <EuiFlexGroup direction="column">
           <EuiFlexItem>
-            <DocViewerLinks hit={hit} indexPattern={indexPattern} columns={columns} />
+            <EuiText size="s">
```

Comment:
callout: placing here instead of inside component to avoid changing font size in expanded view